nixos/modules/programs/dconf.nix

```
@@ -22,18 +33,22 @@ in
       profiles = mkOption {
         type = types.attrsOf types.path;
         default = {};
-        description = "Set of dconf profile files.";
+        description = "Set of dconf profile files, installed at <literal>/etc/dconf/profiles/name<literal>.";
         internal = true;
       };
 
+      packages = mkOption {
+        type = types.listOf types.package;
+        default = [];
+        description = "A list of packages which provide dconf profiles and databases in <literal>$out/etc/dconf</literal>.";
```

Not always the out output.

```suggestion
        description = "A list of packages which provide dconf profiles and databases in <filename>/etc/dconf</filename>.";
```